### PR TITLE
ida extract library funcs identified via flirt

### LINKED ISSUE #658: IDA: extract API features via function names recognized by IDA/FLIRT
from https://github.com/fireeye/capa/issues/657

### PR DESCRIPTION
**Pull request by mr-tz**
closes #658

### Checklist

- [ ] No CHANGELOG update needed

- [x] No new tests needed

- [x] No documentation update needed

### CODE REVIEW

File: capa/features/extractors/ida/insn.py [hunk 1]
```diff
@@ -95,11 +92,29 @@ def extract_insn_api_features(f, bb, insn):
...
+    if not insn.get_canon_mnem() in ("call", "jmp"):
+        return
+
     for api in check_for_api_call(f.ctx, insn):
         dll, _, symbol = api.rpartition(".")
         for name in capa.features.extractors.helpers.generate_symbols(dll, symbol):
             yield API(name), insn.ea
 
+    # extract IDA/FLIRT recognized API functions
+    targets = list(idautils.CodeRefsFrom(insn.ea, False))
```
Comment (mike-hunhoff) [inline capa/features/extractors/ida/insn.py:104]:
```suggestion
    targets = tuple(idautils.CodeRefsFrom(insn.ea, False))
```

File: capa/features/extractors/ida/insn.py [hunk 2]
```diff
@@ -95,11 +92,29 @@ def extract_insn_api_features(f, bb, insn):
...
+    if not targets:
+        return
+
+    target = targets[0]
+    target_func = idaapi.get_func(target)
+    if not target_func or target_func.start_ea != target:
+        # not a function (start)
+        return
+
+    if idaapi.get_func(target).flags & idaapi.FUNC_LIB:
```
Comment (mike-hunhoff) [inline capa/features/extractors/ida/insn.py:114]:
```suggestion
    if target_func.flags & idaapi.FUNC_LIB:
```

File: capa/features/extractors/ida/insn.py [hunk 3]
```diff
@@ -95,11 +92,29 @@ def extract_insn_api_features(f, bb, insn):
...
+        name = idaapi.get_name(target)
```
Comment (mike-hunhoff) [inline capa/features/extractors/ida/insn.py:115]:
```suggestion
        name = idaapi.get_name(target_func.start_ea)
```

File: capa/features/extractors/ida/insn.py [hunk 4]
```diff
@@ -95,11 +92,29 @@ def extract_insn_api_features(f, bb, insn):
     example:
         call dword [0x00473038]
     """
+    if not insn.get_canon_mnem() in ("call", "jmp"):
```
Comment (mike-hunhoff) [inline capa/features/extractors/ida/insn.py:95]: nice catch!

**Review by mike-hunhoff — CHANGES_REQUESTED** [review]
Looks good, thank you for coding this! Some minor changes suggested, otherwise LGTM 🚀

**Comment by mr-tz** [comment]
thanks for the feedback, Mike!

**Pull request merged**